app/recordtransfer/utils.py

```diff
@@ -469,7 +540,73 @@ def _validate_file_extension(filename: str, filesize: int) -> dict:
     return {"accepted": True}
 
 
-def _validate_file_size(filename: str, filesize: int) -> dict:
+def _validate_mime_type(filename: str, filesize: int, file: UploadedFile) -> dict:
+    """Check if the file's MIME type matches the expected MIME type for its extension.
+
+    Only performs validation if file_content is provided and magic library is available.
```

Comment:
This docstring looks outdated:

```suggestion
    Only performs validation if magic library is available.
```

Reply:
Yes, good catch.